src/snowcli/cli/core/__init__.py

```diff
@@ -0,0 +1,11 @@
+import typer
+
+from . import (
+    stage,
+    warehouse,
+)
+
+app = typer.Typer(context_settings={"help_option_names": ["-h", "--help"]})
+
+app.add_typer(stage.app, name="function")
+app.add_typer(warehouse.app, name="package")
```

Comment:
Are those names correct? Shouldn't id be `stage` and `warehouse`?

Reply:
copy paste 😢